include/odp/api/spec/pool.h

```diff
@@ -273,6 +273,42 @@ typedef uintptr_t (*odp_pool_ext_mem_allocate_t)(void *arg, uint32_t total_sz, u
  */
 typedef void (*odp_pool_ext_mem_deallocate_t)(void *arg, uintptr_t ptr);
 
+/**
+ * ODP pool buffer type iterator callback.
+ *
+ * This callback shall be invoked by the implementation when each odp_buffer_t type is freed
+ * to the pool in odp_pool_create().
+ *
+ * @param arg    Opaque pointer registered by the user with odp_pool_param_t::buf::arg
+ * @param data_addr Pointer to the buffer start address.
+ *
+ * On successful pool creation, the application can access the same buffer
+ * address using odp_buffer_addr().
+ *
+ */
+typedef void (*odp_pool_buffer_iterator_t)(void *arg, void *data_addr);
```

Comment:
I would rather call this odp_pool_buffer_iterator_cb_t instead, as it is an iteration callback and not an iterator (an object holding iteration state).

Reply:
Ack